JavaStreams_task1/src/main/java/org/Marta/App.java

```diff
@@ -0,0 +1,16 @@
+package org.Marta;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class App {
+    public static void main(String[] args) {
+        IntStream ints = IntStream.range(-20, 21);
+        List<Integer> filteredNums =  ints
+                .filter(num -> num % 3 == 0)
+                .boxed()
+                .collect(Collectors.toList());
+        filteredNums.forEach(System.out::println);
```

Comment:
Nie widze powodu, aby to tak dzielić na osobne zmienne.

`metoda implementująca wymaganie powinna zwracać kolekcję liczb`

Więc w jednej metodzie:

```
return IntStream.range(from, to)
               .filter(...)
               .boxed()
               .collect([...]);


```

w drugiej:

```
wywolanieMetody(-20,21).forEach(System.out::println);
```